meta-frc971/recipes-devtools/cuda/cuda-nvcc-11-8%.bbappend

```diff
@@ -0,0 +1,13 @@
+SRC_URI += "http://http.us.debian.org/debian/pool/non-free/n/nvidia-cuda-toolkit/nvidia-cuda-toolkit_11.8.89~11.8.0-5~deb12u1_amd64.deb"
```

Comment:
shouldn't this be an arm version of cuda toolkit?  I see it is set to amd64.

Reply:
@AustinSchuh here is the comment that I mentioned

Reply:
This is intentional although very weird. We install these x86 binaries because in our build system they use the rootfs from yocto, and it runs these two binaries on the local machine, not emulating arm64. There is a question to whether this should be the case however it seems to be fine right now.

Reply:
Can you add a comment to this effect?